package.json

```diff
@@ -27,16 +27,16 @@
   },
   "homepage": "https://github.com/itajaja/relay-mock#readme",
   "devDependencies": {
-    "babel-cli": "^6.11.4",
-    "babel-core": "^6.11.4",
-    "babel-eslint": "^6.1.2",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.11.1",
-    "babel-preset-stage-0": "^6.5.0",
-    "rimraf": "^2.5.4"
+    "babel-cli": "^6.24.0",
```

Comment:
can you remove the devDeps upgrades from this PR? I'd rather do it in a different PR, after testing it actually builds